tests/App/Kernel.php

```diff
@@ -89,23 +84,15 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
             'http_method_override' => false,
         ];
 
-        if (class_exists(NativeSessionStorageFactory::class)) {
-            $frameworkConfig['session'] = ['storage_factory_id' => 'session.storage.factory.mock_file'];
-        } else {
-            $frameworkConfig['session'] = ['storage_id' => 'session.storage.mock_file'];
-        }
+        $frameworkConfig['session'] = ['storage_factory_id' => 'session.storage.factory.mock_file'];
 
         $container->loadFromExtension('framework', $frameworkConfig);
 
         $securityConfig = [
             'firewalls' => ['main' => []],
         ];
 
-        if (class_exists(AuthenticatorManager::class)) {
-            $securityConfig['enable_authenticator_manager'] = true;
-        } else {
-            $securityConfig['firewalls']['main']['anonymous'] = true;
-        }
+        $securityConfig['enable_authenticator_manager'] = true;
```

Comment:
You need to do the same if for Sf 6.2 ('enable_authenticator_manager' is deprecated for SF 6.2).

The class_exist you can use IsGranted::class.